CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main / unreleased
 
+* [FEATURE] Distributor: Add experimental Influx handler. #10153
```

Comment:
Please move this to the Grafana Mimir section below.

Reply:
Done.